assets/css/api-tokens.css

```diff
@@ -0,0 +1,17 @@
+#add-api-token-form {
+	flex-direction: row;
+	justify-content: right;
+}
+
+#add-api-token-form > * {
+	align-content: center;
+}
+
+.show-on-click {
+	padding: 0.25rem;
+	opacity: 0;
+}
```

Comment:
this should be styled similar to an input ideally
it's kinda weird that it's just invisible when not shown
you don't really know where to click

Reply:
but you cannot input anything, seems confusing to me.
alternatively I can try to show a hover effect ?

Reply:
You can think of it as a disabled input